drivers/audio/cxd56_src.c

```diff
@@ -0,0 +1,606 @@
+/****************************************************************************
+ * drivers/audio/cxd56_src.c
+ *
+ *   Copyright 2020 Sony Semiconductor Solutions Corporation
```

Comment:
Can these files come in with the Apache header?

Reply:
@btashton they will all be migrated to Apache once SGA is in place.

Reply:
> @btashton they will all be migrated to Apache once SGA is in place.

For all other PRs, we have insisted that the Apache license be in place prior to bringing the code into the repository.  I don't doubt for a minute that the headers will be updated if/when the SGA is in place.  But we need  apply this rule consistently to all code.

Perhaps we need to clarification?  Perhaps we should wait until the SGA is in place?

@justinmclean Do you have any guidance on this?

Reply:
@patacongo the SGA is on its way but since it needed to be changed it will take some more time

Reply:
I would be OK on this but only because I have seen the discussion about getting paperwork sorted out and trust that this will be resolved. We have pushed back on other submissions so I agree about needing to be consistent and fair here.

If for some reason the paperwork fell through this would just end up in the same place as other BSD code and we would have to call it out in the NOTICE file. That would be unfortunate, but not a problem.